### DISCUSSION TITLE
v2.2.2

### DISCUSSION THREAD

**Discussion by crohr** [runs-on/runs-on#70] [Announcements]
## New regions

London (`eu-west-2`) and Tokyo (`ap-northeast-1`) are now supported regions!

## Fixes

- Adds `runs-on-repo-full-name` tag to instances. Fixes #67.
- No longer require public access on the S3 bucket `agents/` subpath. Should help with compliance in certain settings.
- Upgrade dependencies, fix potential XSS issue in setup page when entering malicious input as an organisation name in the CF template.
- Switch to distroless container image for Server instead of debian.
- Reduce scope for `ec2:Describe*` permission.
- Fix handling of `admins` in configuration file, and `DefaultAdmins` parameters. Previously `DefaultAdmins` was not properly taken into account. And `admins` from the configuration file were added to the repo collaborators with admin access, while it should have override them (this was lost during the nodejs => go transition).

## Base AMIs

- Added `ssh` cloud-init module back. Should help for users that extend the base AMIs with packer, since it needs to store an ephemeral ssh key in the instance.
- Plan is to have the base AMIs released on a regular schedule, every 15 days (instead of whenever GitHub publishes a new release), to keep up with security patches.

## Request for comments / possible deprecations for next version

- Set SSH access to disabled by default?

## Upgrade

CloudFormation template for upgrade:

* latest: https://runs-on.s3.eu-west-1.amazonaws.com/cloudformation/template.yaml
* v2.2.2: https://runs-on.s3.eu-west-1.amazonaws.com/cloudformation/template-v2.2.2.yaml

<hr /><em>This discussion was created from the release <a href='https://github.com/runs-on/runs-on/releases/tag/v2.2.2'>v2.2.2</a>.</em>

**Comment by rathpc** [comment]
> Set SSH access to disabled by default?

That seems like a good idea to me!
